SPECS/td-agent/rdkafka.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/ext/Rakefile b/ext/Rakefile
+index 5e946f2..f4e992f 100644
+--- a/ext/Rakefile
++++ b/ext/Rakefile
+@@ -30,7 +30,7 @@ task :default => :clean do
+   end
+ 
+   recipe.files << {
+-    :url => "https://codeload.github.com/edenhill/librdkafka/tar.gz/v#{Rdkafka::LIBRDKAFKA_VERSION}",
++    :url => "file:///usr/src/mariner/SOURCES/librdkafka-1.4.0.tar.gz",
```

Comment:
This takes a dependency on  implementation details of the build system. The file path `/usr/src/mariner` is the RPM build root chosen by `pkgworker`. I don't believe we should be taken an explicit dependency on this, can this path be made relative instead?

Reply:
This file belonged to the previous implementation of td-agent and is no longer needed in the current implementation. It was pushed by mistake. Will take it out of the PR.

Reply:
Resolved